pkg/api/common/versions.go

```diff
@@ -88,8 +88,9 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.11.5":         false,
 	"1.11.6":         false,
 	"1.11.7":         false,
-	"1.11.8":         true,
+	"1.11.8":         false,
```

Comment:
Does this change preclude 1.11.8 from Azure Stack as well?

Reply:
Preclude is the wrong verb. Will Azure Stack scenarios still be able to build 1.11.8 clusters (using their own VHD)?

Reply:
Good point. If we want to keep Azure Stack 1.11.8 support until 1.11.10 is available we shouldn't disable it.

Reply:
(the answer to your question is no,  Azure Stack scenarios will not be able to build 1.11.8 clusters with this change).

Reply:
Great, yeah, let's undo this. In order to maintain non-Azure Stack velocity, let's be more permissive in allowing > 2 patch versions so that non-Azure Stack can move ahead quickly and not block on Azure Stack.

Reply:
I left 1.11.8 enabled for now and created #1219 so we remove it later when AzureStack support for 1.11.10 has been finalized.